content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,112 @@
+---
+title: BigQuery Cost Allocation
+description: Learn how to allocate Cloud Cost Management spending across your organization with BigQuery Cost Allocation.
+further_reading:
+  - link: "/cloud_cost_management/"
+    tag: "Documentation"
+    text: "Learn about Cloud Cost Management"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates the costs of your Google BigQuery resources to individual queries and workloads. Use cost metrics enriched with tags from queries, projects, and reservations to visualize BigQuery workload costs in the context of your entire cloud bill.
+
+## Resources
+CCM allocates costs for BigQuery resources for both types of Analysis costs (on-demand and reservation-based). CCM displays costs for resources including query-level analysis, storage, and data transfer on the [**BigQuery** dashboard][1].
+
+## Prerequisites
+
+The following table presents the list of collected features and the minimal requirements:
+
+| Feature | Requirements |
+|---|---|
+| Retrieve tags from labels of a query | Supported without monitoring or reservations |
+| Query-Level Cost Attribution | BigQuery monitoring enabled |
+| Reservation Cost Allocation | BigQuery reservations configured |
+
+1. Configure the Google Cloud Cost Management integration on the [Cloud Cost Setup page][2].
+2. Enable BigQuery monitoring in your Google Cloud project.
+3. For reservation cost allocation, configure BigQuery reservations in your project.
```

Comment:
What does a user need to explicitly do to set up the bigquery logs? We should either list out the setup exactly or link to a page in the docs and say set this up